src/components/search/SuggestionProductCard/suggestion-product-card.module.scss

```
@@ -0,0 +1,60 @@
+@import "src/styles/scaffold";
+
+.fs-suggestion-product-card[data-fs-suggestion-product-card] {
```

Do we need to double the specificy? does it work only with the class?

```suggestion
.fs-suggestion-product-card {
```